examples/web/src/components/Root/index.js

```diff
@@ -7,11 +7,12 @@ import BlogList from 'components/BlogList'
 import Post from 'components/Post'
 import Blog from 'components/Blog'
 import ModerationQueue from 'components/ModerationQueue'
+import logoSrc from './WatermelonLogo.svg'
 
 import style from './style'
 
 class Root extends Component {
-  state = { isGenerating: false }
+  state = { isGenerating: false, isGenerated: false }
```

Comment:
nope, `isGenerated: false` is not necessary, and breaks the app after you refresh. Look at examples/native. We just need to stop rendering the lists during generation